frontend/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Does this solve the warning that gets thrown when you run the app?

Reply:
babel.config.js and jest.config.js and config files used in order to get the npm test in react-testing library to work correctly. I have no idea how to get rid of the warnings to be honest